.github/workflows/changelog_check.yml

```diff
@@ -0,0 +1,12 @@
+name: Changelog check
+
+on:
+  pull_request:
+    branches: master
+
+jobs:
+  diff:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: bash .github/scripts/changelog_checker
```

Comment:
The script says `#!/bin/sh`, why is `bash` needed here ?
(Ok, we probably don't care ;) )

Reply:
Ah my bad, this is specific for GH runners with a limited choice, not any Docker image like GL ones, so suggesting Alpine makes no sense here

Reply:
No, just a habit to begin my scripts with that shebang